🚨 Beta IDE refresh - to merge on Aug 16th

mirnawong1 opened the pull request:
IDE Beta v1.1 documentation and subsection under 'dbt Cloud IDE' section

## Description & motivation
The purpose of this PR is to create content aligning with the IDE v1.1 Beta launch on August 16th and set up the foundation for documentation once the new IDE goes GA nearer Oct.

## To-do before merge
Include any notes about things that need to happen before this PR is merged on Aug 16th only!!
- [x] Change the base branch
- [x] Leona will remove any extra content from next branch
- [x] Ensure [release.md](https://github.com/dbt-labs/docs.getdbt.com/blob/beta-ide-refresh/website/docs/docs/dbt-cloud/release-notes/07-Aug-2022/ide-improvement-beta.md) file is included in pr1818 !
- [x] Add updated IDE v1.1 images to the beta.md file on Aug 16th!
- [x] Double-check re-directs and links on the page  on Aug 16th!
- [x] Content approval ([Google Docs](https://docs.google.com/document/d/1ds9McWyrwLq-j-lcr4gPFoldphaiBRaRLSB1_hKDf4c/edit#heading=h.tpm0h48c2g1b)
- [x] Share staging link with Nate and IDE refresh team  on Aug 16th!

Tasks already completed:
Issue# (will create one PR for this)
☑️ Restructure docs
☑️ Update [.md files](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-cloud/cloud-ide)
☑️ Remove old IDE images and videos
☑️ When possible, replace short Looms with text and sometimes images.
☑️ Add new content/sections
☑️ Update sidebar.js file

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

Notion Docs Project: 
https://www.notion.so/dbtlabs/IDE-v1-1-Documentation-f8a3b1afec1b41e9a47359804933cb74
[Docs Project](https://www.notion.so/dbtlabs/IDE-v1-1-Beta-f8a3b1afec1b41e9a47359804933cb74)

mirnawong1 committed:
Update the-dbt-ide.md

Replacing content in 'dbt-ide' file.

mirnawong1 committed:
Create mult-selector.mp4

mirnawong1 committed:
Delete mult-selector.mp4

mirnawong1 committed:
multi-selector.mp4

mirnawong1 committed:
Delete giphy (1).mp4

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Delete multi-selector.mp4

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Delete multi-selector.gif

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Delete git overview.jpg

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Create learn-dbt-ide.md

added learn dbt ide section with images

mirnawong1 committed:
Update and rename viewing-docs-in-the-ide.md to use-cloud-ide.md

mirnawong1 committed:
Update and rename the-ide-git-button.md to version-control-ide.md

mirnawong1 committed:
Update _redirects

change redirect links

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update learn-dbt-ide.md

mirnawong1 committed:
Update version-control-ide.md

mirnawong1 committed:
Update sidebars.js

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update and rename version-control-ide.md to the-ide-git-button.md

mirnawong1 committed:
Update and rename use-cloud-ide.md to viewing-docs-in-the-ide.md

mirnawong1 committed:
Update and rename learn-dbt-ide.md to ide-beta.md

added draft of new beta ide docs with callout

mirnawong1 committed:
Update _redirects

mirnawong1 committed:
Update sidebars.js

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

added beta banner

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Create 16-Aug-2022

mirnawong1 committed:
Delete 16-Aug-2022

mirnawong1 committed:
Create ide-improvement-beta.md

adding ide beta release notes
need to update the version

mirnawong1 committed:
Update ide-improvement-beta.md

updated content corrections

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

revised beta info callout

mirnawong1 committed:
Update ide-beta.md

updated callout with grammar

mirnawong1 committed:
Update the-dbt-ide.md

JKarlavige commented:
@runleonarun before i continue going through and reverting these commits, does the latest two commits from me look to be reverting the correct code?

mirnawong1 committed:
Rename website/docs/docs/dbt-cloud/release-notes/16-Aug-2022/ide-improvement-beta.md to website/docs/docs/dbt-cloud/release-notes/07-Aug-2022/ide-improvement-beta.md

updated order of folder

mirnawong1 committed:
Update ide-improvement-beta.md

mirnawong1 committed:
Update ide-improvement-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

removed pre-requisite callout and added as a sentence in the body. this is to avoid having too many callouts and also have the pre-requisites flow

mirnawong1 committed:
Update ide-improvement-beta.md

mirnawong1 committed:
Update ide-improvement-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update the-dbt-ide.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Delete command-bar.png

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Delete command_bar.png

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Update ide-beta.md

updated with ly's content edits and updated screenshots too

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-improvement-beta.md

updated based on Ly's content review https://docs.google.com/document/d/1ds9McWyrwLq-j-lcr4gPFoldphaiBRaRLSB1_hKDf4c/edit#

mirnawong1 committed:
Add files via upload

mirnawong1 committed:
Update ide-beta.md

updated changes based on Ly's feedback https://docs.google.com/document/d/1ds9McWyrwLq-j-lcr4gPFoldphaiBRaRLSB1_hKDf4c/edit#

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-beta.md

mirnawong1 committed:
Update ide-improvement-beta.md

runleonarun reviewed (approved):
Let's ship and fix images after ship!
